package.json

```diff
@@ -56,6 +56,7 @@
     "karma-mocha": "^1.3.0",
     "karma-ng-html2js-preprocessor": "^1.0.0",
     "karma-spec-reporter": "0.0.32",
+    "minimist": "^1.2.0",
```

Comment:
I think this can be in `scripts/package.json` instead of the root folder?

Reply:
Agree, it should be moved there